src/module/Subject/src/Subject/Manager/SubjectManager.php

```diff
@@ -121,12 +121,12 @@ protected function getEntities(TaxonomyTermInterface $term)
         return $term->getAssociatedRecursive('entities');
     }
 
-    protected function normalize(EntityInterface $entity, Collection $collection)
+    protected function isRevised(EntityInterface $entity, Collection $collection)
     {
-        $normalized = $this->normalizer->normalize($entity->getHead());
-        $collection->set(- $normalized->getMetadata()
-            ->getCreationDate()
-            ->getTimestamp(), $normalized);
+        if ($entity->isUnrevised() && !$collection->contains($entity)) {
```

Comment:
Since all children of the entities pass through here, also ones which aren't unrevised, I went back to the logic we had here before.